capstone/src/main/java/com/google/univiz/api/SuggestionData.java

```diff
@@ -1,4 +1,20 @@
 package com.google.univiz.api;
 
 /** THIS IS A DUMMY CLASS TO ALLOW COMPILATION: REMOVE BEFORE MERGING */
-public class SuggestionData {}
```

Comment:
ditto about ```AutoValue``` here

Reply:
ah ok-should I go through the work of doing autovalue stuff here since this is purely for compilation?

Reply:
no, but if you're going to have this class in another PR, just remember to use ```AutoValue```